src/go/k8s/controllers/redpanda/console_controller.go

```diff
@@ -65,13 +65,13 @@ const (
 	NoSubdomainEvent = "NoSubdomain"
 )
 
-//+kubebuilder:rbac:groups=apps,namespace=default,resources=deployments,verbs=get;list;watch;create;update;patch;delete
-//+kubebuilder:rbac:groups=apps,namespace=default,resources=configmaps,verbs=get;list;watch;create;update;patch;delete
-//+kubebuilder:rbac:groups=core,namespace=default,resources=events,verbs=get;list;watch;create;update;patch
+//+kubebuilder:rbac:groups=apps,resources=deployments,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=apps,resources=configmaps,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=core,resources=events,verbs=get;list;watch;create;update;patch
 
-//+kubebuilder:rbac:groups=redpanda.vectorized.io,namespace=default,resources=consoles,verbs=get;list;watch;create;update;patch;delete
-//+kubebuilder:rbac:groups=redpanda.vectorized.io,namespace=default,resources=consoles/status,verbs=get;update;patch
-//+kubebuilder:rbac:groups=redpanda.vectorized.io,namespace=default,resources=consoles/finalizers,verbs=update
```

Comment:
This need to be reverted. This will generate `ClusterRole` where we need `Role`

Reply:
This is for console controller, this is required.